tests/stc_containers/leaks_check.py

```diff
@@ -0,0 +1,62 @@
+
+# pylint: disable=missing-function-docstring, missing-module-docstring
+def create_list():
+    a = [1,2,3]
+
+def create_set():
+    a = {1,2,3}
+
+def create_dict():
+    a = {1:1,2:2,3:3}
+
+def list_reassign():
+    a = [1,2,3]
+    a = [1,2,3,4]
+    a = [i for i in range(15)] # pylint: disable=unnecessary-comprehension
```

Comment:
Maybe we can just avoid the pylint error?
```suggestion
    a = [i*2 for i in range(15)] # pylint: disable=unnecessary-comprehension
```